app/retail/templates/shared/footer_scripts.html

```diff
@@ -15,35 +15,34 @@
     along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 {% endcomment %}
-{% load static %}
+{% load static compress %}
 <script type="text/javascript">
   var static_url = "{{ STATIC_URL }}";
   var media_url = "{{ MEDIA_URL }}";
   document.contxt = {{ json_context|safe }};
 </script>
-<script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
-<script src="{% static "v2/js/lib/jquery.js" %}"></script>
 {% if user.is_authenticated and env == 'prod' %}
-  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
-{% elif user.is_authenticated %}
-  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/vue"></script> 
 {% endif %}
+
 {% include 'shared/sentry.html' %}
-<script src="{% static "v2/js/lib/popper.min.js" %}"></script>
-<script src="{% static "v2/js/lib/bootstrap.min.js" %}" crossorigin="anonymous"></script>
 <script>
   const bootstrapModal = $.fn.modal.noConflict()
```

Comment:
this is going to give `$.fn.modal is undefined` as bootstrap is loaded after this declaration.